QuantExt/qle/termstructures/crossccybasisswaphelper.hpp

```diff
@@ -59,7 +59,12 @@ class CrossCcyBasisSwapHelper : public RelativeDateRateHelper {
                             const Handle<YieldTermStructure>& spreadDiscountCurve, bool eom = false,
                             bool flatIsDomestic = true,
                             boost::optional<QuantLib::Period> flatTenor = boost::none,
-                            boost::optional<QuantLib::Period> spreadTenor = boost::none);
+                            boost::optional<QuantLib::Period> spreadTenor = boost::none,
```

Comment:
I added the shown 5 parameters as optional at the end of the list in order to keep the constructor backward compatible.